apps/mobile-alphabet/src/app/Components/Menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Text, View } from 'react-native';
+
+interface AlphabetData {
+    data: {
+        name: string;
+        name_english: string;
+        poster: {
+            name: string;
+            url: string;
+        };
+    };
+}
+
+export function MenuScreen() {
+    const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
```

Comment:
You need a representation of `error` state. Maybe 
```ts
AlphabetData | null | Error
```

Note that you may also want a separate notion of `pending` to be explicit about loading.